public/app/features/alerting/unified/components/rule-editor/alert-rule-form/simplifiedRouting/SimplifiedRouting.tsx

```diff
@@ -1,46 +1,19 @@
-import { css } from '@emotion/css';
-import { createAction, createReducer } from '@reduxjs/toolkit';
-import React, { useEffect, useReducer } from 'react';
+import React, { useMemo } from 'react';
 import { useFormContext } from 'react-hook-form';
 
-import { GrafanaTheme2 } from '@grafana/data';
-import { Icon, Link, Stack, Text, useStyles2 } from '@grafana/ui';
 import { AlertmanagerProvider } from 'app/features/alerting/unified/state/AlertmanagerContext';
 import { RuleFormValues } from 'app/features/alerting/unified/types/rule-form';
-import {
-  AlertManagerDataSource,
-  getAlertManagerDataSourcesByPermission,
-} from 'app/features/alerting/unified/utils/datasource';
-import { createUrl } from 'app/features/alerting/unified/utils/url';
+import { getAlertManagerDataSourcesByPermission } from 'app/features/alerting/unified/utils/datasource';
 
-import { ContactPointSelector } from './ContactPointSelector';
+import { AlertManagerManualRouting } from './AlertManagerRouting';
 
-export interface AMContactPoint {
-  alertManager: AlertManagerDataSource;
-  selectedContactPoint?: string;
+export interface SimplifiedRoutingProps {
+  toggleOpenRoutingSettings: (nextValue: boolean) => void;
+  isOpenRoutingSettings: boolean;
 }
 
-export const selectContactPoint = createAction<{ receiver: string | undefined; alertManager: AlertManagerDataSource }>(
-  'simplifiedRouting/selectContactPoint'
-);
-
-export const receiversReducer = createReducer<AMContactPoint[]>([], (builder) => {
-  builder.addCase(selectContactPoint, (state, action) => {
-    const { receiver, alertManager } = action.payload;
```

Comment:
we get rid of this actions/reducer as it only sets the fields in the form. React hoook form let us to handle this actions easily accessing the fields with registering/watching with: 
` {...register(`contactPoints.${alertManager}.selectedContactPoint`, { required: true })}`
`watch(`contactPoints.${alertManager}.selectedContactPoint`)`

Reply:
Love this solution!